pkgs/kikit/solidpython/default.nix

```diff
@@ -62,13 +41,18 @@ in
       hash = "sha256-3fJta2a5c8hV9FPwKn5pj01aBtsCGSRCz3vvxR/5n0Q=";
     };
 
+    patches = [
+      # Unpin PrettyTable, see <https://github.com/SolidCode/SolidPython/issues/207>.
+      ./prettytable.patch
+    ];
```

Comment:
We learned recently that nixpkgs actually has a more idiomatic way of relaxing dependencies. See https://nixos.org/manual/nixpkgs/stable/#using-pythonrelaxdepshook, and us using it here: https://github.com/NixOS/nixpkgs/pull/249464/commits/38a7f42312ad59e7604dc1ed01ebc1acaf59849c.

Reply:
Nice, thanks! This might come in handy also for packaging pretalx. I'll update this PR ASAP.

Reply:
Fixed in 35b6a22.